tests/feature/test_wrong.py

```diff
@@ -38,7 +39,7 @@ def then_nevermind():
 )
 @pytest.mark.parametrize('strict_gherkin', [True, False])
 @pytest.mark.parametrize('multiple', [True, False])
-@mock.patch('pytest_bdd.fixtures.pytestbdd_strict_gherkin', autospec=True)
+@mock.patch.object(sys.modules['pytest_bdd.scenario'], 'get_strict_gherkin', return_value=True)
```

Comment:
I never saw patching sys.module directly. Can't we instead use something like
`@mock.patch.object('pytest_bdd.scenario.get_strict_gherkin')` instead?

Actually, if we are going to implement `strict_gherkin` as a parameter for `scenario`, then we don't even need to patch.

Reply:
That was something new to me as well. It is there because of having `__all__` in `__init__.py` file that exposes only `scenario` function and not module.
To be able to access `scenario` module and mock one of its function you have to access it through `sys.modules`.

https://stackoverflow.com/questions/32026987/mock-patch-with-init-py

As you said, we don't need to mock it if we can pass `strict_gherkin` as parameter.